app/src/app/account/credits/available-credits-banner.tsx

```diff
@@ -0,0 +1,33 @@
+import { env } from "~/env.mjs";
+import { getUserFromSession } from "~/lib/session";
+
+import { InfoTooltip } from "~/components/info-tooltip";
+import { viewCall } from "~/lib/fetching";
+import type { HorizonProject } from "~/lib/validation/projects";
+
+export default async function AvailableCreditsBanner() {
+  const user = await getUserFromSession();
+
+  const horizonProfile = await viewCall<HorizonProject>(
+    env.NEXT_PUBLIC_CONTRACT_ACCOUNT_ID,
+    "get_project",
+    { account_id: user?.accountId }
+  );
```

Comment:
I would use an existing function to fetch the project here. I would also redirect to the login route if there is no user like seen in the dashboard page.